diagnostic.md

```diff
@@ -71,13 +71,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+Relative. The path provided only references a single folder rather than the entire path starting from root.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+Absolute is best. This way the file can be read from any directory. 
```

Comment:
In this case relative may be easier because the image file lives inside our current working directory, project. As a result, an absolute path is not needed to access it.